cmd/kola/spawn.go

```diff
@@ -38,19 +42,30 @@ var (
 
 	spawnNodeCount      int
 	spawnUserData       string
+	spawnDetach         bool
 	spawnShell          bool
 	spawnRemove         bool
 	spawnVerbose        bool
 	spawnMachineOptions string
+	spawnSetSSHKeys     bool
+	spawnSSHKeys        []string
 )
 
 func init() {
+	defaultKeys, err := defaultSSHKeyPaths()
+	if err != nil {
+		plog.Warning(err)
+	}
+
 	cmdSpawn.Flags().IntVarP(&spawnNodeCount, "nodecount", "c", 1, "number of nodes to spawn")
 	cmdSpawn.Flags().StringVarP(&spawnUserData, "userdata", "u", "", "file containing userdata to pass to the instances")
+	cmdSpawn.Flags().BoolVarP(&spawnDetach, "detach", "t", false, "-v --shell=false --remove=false, plus -k if -u not set")
```

Comment:
I can see how this flag is useful, but the `-k` conditionally defaulting is kinda weird. Maybe `-k` should always default true, and have the restriction of conflicting with `-u` removed?

I'm not going to argue particularly strongly since it only seems a little odd